mod/01_rating/set/right/variables.rb

```diff
@@ -87,7 +87,15 @@ def default_edit_args args
   end
 
   def metric_list
```

Comment:
I know this is old code, but I might consider a better name for this, like "variable_options".